background/background.js

```diff
@@ -29,15 +29,21 @@ const LivemarkUpdater = {
   },
   async updateAllLivemarks({changedKeys = []} = {}) {
     const livemarks = await LivemarkStore.getAll();
-    for (const feed of livemarks) {
-      try {
+
+    const next = () => {
+      const feed = livemarks.pop();
+      if (feed) {
         this.updateLivemark(feed, {
           forceUpdate: changedKeys.includes(feed.id),
-        });
-      } catch (e) {
-        console.log("Error getting feed", e);
+        }).finally(next);
       }
     }
+
+    livemarks.splice(0, 10).forEach(feed => {
```

Comment:
Can you add a comment ?
```suggestion
    // Only update at most 10 livemarks concurrently
    livemarks.splice(0, 10).forEach(feed => {
```